src/renderer/theme.tsx

```diff
@@ -20,6 +20,9 @@ const theme = responsiveFontSizes(
       },
     },
     typography: {
+      body1: {
```

Comment:
what is this for?

Reply:
Defaults text to light grey instead of black